docker-compose.yml

```diff
@@ -36,6 +36,7 @@ services:
         091e3bedc85447ef936e82bcda88fcac"
       AIRFLOW_CONN_SWIFT_DEFAULT: "s3://vsd_user:${VSD_PASSWD}@\
         4028c44d91dc48b8990069433c203c1f"
+      AIRFLOW_CONN_GOB_GRAPHQL: "https://acc.api.data.amsterdam.nl/https"
```

Comment:
Vraag: Wat is ook al weer de reden om een connectie in de docker-compose.yml op te nemen i.p.v. in de connection store in de Airflow applicatie zelf?

Reply:
Dat past het beste bij de Openstack aanpak, waarbij we alles aansturen obv. env. variabelen.

Reply:
Het is nodig om verschil te kunnen  kunnen maken tussen acceptatie en productie. 
Soms is er ook een andere variabele waaruit  je acceptatie of productie kunt halen, en dan kun je dat met een "if acceptatie" .. in de code doen. Maar zo is het meer configureerbaar als er ooit een development versie komt

Reply:
Ja, goed punt. In Openstack is daar een handige variabele voor ontdekte ik, dus die gebruik ik tegenwoordig:

    AIRFLOW_CONN_GOB_GRAPHQL: "https://{{ acc_dot }}api.data.amsterdam.nl/https"

Wellicht ook zoiets doen in de lokale docker_compose?